pkg/config/validation/validation.go

```diff
@@ -934,13 +931,6 @@ var ValidateEnvoyFilter = registerValidateFunc("ValidateEnvoyFilter",
 		return errs.Unwrap()
 	})
 
-func validateListenerMatchName(name string) error {
-	if newName, f := xds.ReverseDeprecatedFilterNames[name]; f {
```

Comment:
shouldn't we still warn?

Reply:
Will add

Reply:
Do we have to? Those names are no longer relevant and have been removed from Envoy long back. At some point we should clean this up.

Reply:
Why not warn though? If a user is using something we _know_ is wrong, seems warning is strictly beneficial?

We can clean up the _support_ for it, but still warn

Reply:
I think it's better to warn(maybe a metric) for 1 or 2 release before removed

Reply:
>I think it's better to warn(maybe a metric) for 1 or 2 release before removed

This warning is there for multiple releases. Sure. I can add the warning back